src/app/examples/list-example/base-list.component.ts

```diff
@@ -1,12 +1,17 @@
 export class BaseListComponent {
-    myItems = [
-        {title: 'Vestas Wind Systems has a very long name', subTitle: '2000 stk', amount: '5.587.218.309 DKK', detail: 225},
-        {title: 'A.P. Møller-Mærsk', subTitle: '2 stk', amount: '14.019 DKK', detail: 0},
-        {title: 'Novo Nordisk A/S B', subTitle: '18 stk', amount: '7560 DKK', detail: 171},
-        {title: 'Danske Bank A/S', subTitle: '10 stk', amount: '1560 DKK', detail: -171},
-    ];
+  myItems = [
+    {
```

Comment:
Why did prettier only format one of the json objects? or was it done manually.

Reply:
I has actually formatted "all".

Prettier's philosophy is to "cram as much stuff into one line as possible", but if all properties of a JS-object can't be on a single line, it'll nicely format it (by having one property per line).

Reply:
Understood, so this is the Prettier way of doing. Let follow that 😄